src/main/scala/nodes/learning/KernelRidgeRegression.scala

```diff
@@ -0,0 +1,343 @@
+package nodes.learning
+
+import workflow.LabelEstimator
+
+import breeze.linalg._
+import breeze.math._
+import breeze.numerics._
+import org.apache.spark.rdd.RDD
+import org.apache.spark.broadcast.Broadcast
+import org.apache.spark.SparkConf
+import org.apache.spark.SparkContext
+import org.apache.spark.SparkContext._
+import utils._
+
+
+import edu.berkeley.cs.amplab.mlmatrix.RowPartitionedMatrix
+import edu.berkeley.cs.amplab.mlmatrix.util.{Utils => MLMatrixUtils}
+
+import scala.util.Random
+
+class KernelRidgeRegression(
+  kernelBlockGenerator: KernelGenerator,
```

Comment:
Similar comment as in the kernelModel - I think we can make the `kernel` transformation a part of the pipeline and then this solver is just a BCD based solver for `Kx = b` where `K` is the input to this estimator.

@etrain Does this sound good to you ?